components/lib/editor/Editor.js

```diff
@@ -5,6 +5,7 @@ import { useMergeProps, useMountEffect, useUpdateEffect } from '../hooks/Hooks';
 import { DomHandler, classNames } from '../utils/Utils';
 import { EditorBase } from './EditorBase';
 import { useRef } from 'react';
+import 'quill/dist/quill.snow.css';
```

Comment:
Remove this and copy the CSS into the block below

Reply:
Updated